app/models/manageiq/providers/azure/cloud_manager/vm.rb

```diff
@@ -1,7 +1,9 @@
 class ManageIQ::Providers::Azure::CloudManager::Vm < ManageIQ::Providers::CloudManager::Vm
+  include_concern 'Operations'
+
   def provider_object(connection = nil)
     connection ||= ext_management_system.connect
-    connection.instances[ems_ref]
+    connection
```

Comment:
We haven't encapsulated the results of requests into bonafide Ruby objects yet. So right now it's VMService#start(some_vm, some_group) rather than VM#start.

Still, I can refactor this a bit. Let me see how far I can bend it to fit the approach the others are using.